src/components/Navigation.astro

```diff
@@ -21,26 +23,33 @@ const featuredSponsors = DATA.sponsors.filter(
         height="40"
         width="200"
         loading="eager"
-        />
+      />
     </a>
   </div>
-  <svg class="h-6 w-6 cursor-pointer xl:hidden block"> </svg>
-  <button id="menu-button" class="aspect-square block xl:hidden">
-    <i class="ph ph-list text-3xl"></i>
-  </button>
   <div
-    class="hidden py-3 xl:py-0 w-full xl:flex xl:items-center xl:w-auto"
     id="menu"
+    class="
+    w-full md:w-fot max-w-[700px]
+    h-full
+    max-xl:fixed max-xl:top-0 max-xl:right-0 max-xl:z-40 max-xl:p-4
+    max-xl:overflow-y-auto
+    transition-transform max-xl:translate-x-full
+    max-xl:bg-white max-xl:dark:bg-dark-background
+    max-xl:duration-0"
+    tabindex="-1"
   >
-    <ul
-      class="flex justify-evenly gap-2 text-sm flex-col lg:flex-row font-bold"
+  <ul
+      class="flex justify-evenly
+      xl:gap-2 max-xl:gap-10
```

Comment:
I think you want to use gap-2 xl:gap-10
![image](https://github.com/user-attachments/assets/74120768-ca4b-46bd-b80a-3b4256b667dc)